oauth-common/src/main/java/io/strimzi/kafka/oauth/validator/BackOffTaskScheduler.java

```diff
@@ -34,50 +44,50 @@
     private final ScheduledExecutorService service;
     private final Runnable task;
 
-    private int minPauseSeconds = 1;
-    private int maxIntervalSeconds;
-    private int cutoffIntervalSeconds;
+    private final int minPauseSeconds;
+    private final int cutoffIntervalSeconds;
 
     // Only one task is scheduled at a time
-    private AtomicBoolean taskScheduled = new AtomicBoolean(false);
+    private final AtomicBoolean taskScheduled = new AtomicBoolean(false);
 
     private long lastExecutionAttempt;
```

Comment:
Should this be `volatile`? It's written when running that task and read in `scheduleTask()`, which is called from `io.strimzi.kafka.oauth.validator.JWTSignatureValidator#validate` which I think is running in a different thread.

Reply:
I don't think it's a problem, since scheduleTask() method isn't cycling in a loop it needs to break out of based on shared variable value. Even if I'm wrong about that and on super rare occasion it would happen that scheduleTask() would not see the latest updated lastExecutionAttempt, then the pause before the next execution would just be shorter than otherwise.